### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by sijad**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,16 +11,10 @@ This file is generated by `gen_docker_fish_completions.py` from:
 
 With [Fisherman]
 
-```
+```fish
```
Comment (bucaran) [inline README.md]: No need for fish here.

File: README.md [hunk 2]
```diff
@@ -11,16 +11,10 @@ This file is generated by `gen_docker_fish_completions.py` from:
...
 fisher i fishery/docker-completion
```
Comment (bucaran) [inline README.md]: Let's make all the install instructions simply:

If the plugin is in the fisherman org:

```
fisher plugin
```

If it's outside

```
fisher namespace/plugin
```

**Comment by sijad** [comment]
is this ok?

**Comment by bucaran** [comment]
Almost, just make fisherman always lowercase ;)

**Comment by sijad** [comment]
sure

**Pull request merged**